README.md
LonelyTwitter
=============


changes.txt
diff --git a/LonelyTwitter/bin/LonelyTwitter.apk b/LonelyTwitter/bin/LonelyTwitter.apk
index e73c492..84a87c3 100644
Binary files a/LonelyTwitter/bin/LonelyTwitter.apk and b/LonelyTwitter/bin/LonelyTwitter.apk differ
diff --git a/LonelyTwitter/bin/classes.dex b/LonelyTwitter/bin/classes.dex
index a769af7..4066529 100644
Binary files a/LonelyTwitter/bin/classes.dex and b/LonelyTwitter/bin/classes.dex differ
diff --git a/LonelyTwitter/bin/jarlist.cache b/LonelyTwitter/bin/jarlist.cache
deleted file mode 100644
index 0565465..0000000
--- a/LonelyTwitter/bin/jarlist.cache
+++ /dev/null
@@ -1,3 +0,0 @@
-# cache for current jar dependency. DO NOT EDIT.
-# format is <lastModified> <length> <SHA-1> <path>
-# Encoding is UTF-8
diff --git a/LonelyTwitter/bin/res/crunch/drawable-hdpi/ic_launcher.png b/LonelyTwitter/bin/res/crunch/drawable-hdpi/ic_launcher.png
deleted file mode 100644
index 882eb14..0000000
Binary files a/LonelyTwitter/bin/res/crunch/drawable-hdpi/ic_launcher.png and /dev/null differ
diff --git a/LonelyTwitter/bin/res/crunch/drawable-ldpi/ic_launcher.png b/LonelyTwitter/bin/res/crunch/drawable-ldpi/ic_launcher.png
deleted file mode 100644
index 18689f6..0000000
Binary files a/LonelyTwitter/bin/res/crunch/drawable-ldpi/ic_launcher.png and /dev/null differ
diff --git a/LonelyTwitter/bin/res/crunch/drawable-mdpi/ic_launcher.png b/LonelyTwitter/bin/res/crunch/drawable-mdpi/ic_launcher.png
deleted file mode 100644
index 02e96b9..0000000
Binary files a/LonelyTwitter/bin/res/crunch/drawable-mdpi/ic_launcher.png and /dev/null differ
diff --git a/LonelyTwitter/bin/res/drawable-hdpi/ic_launcher.png b/LonelyTwitter/bin/res/drawable-hdpi/ic_launcher.png
new file mode 100644
index 0000000..882eb14
Binary files /dev/null and b/LonelyTwitter/bin/res/drawable-hdpi/ic_launcher.png differ
diff --git a/LonelyTwitter/bin/res/drawable-ldpi/ic_launcher.png b/LonelyTwitter/bin/res/drawable-ldpi/ic_launcher.png
new file mode 100644
index 0000000..18689f6
Binary files /dev/null and b/LonelyTwitter/bin/res/drawable-ldpi/ic_launcher.png differ
diff --git a/LonelyTwitter/bin/res/drawable-mdpi/ic_launcher.png b/LonelyTwitter/bin/res/drawable-mdpi/ic_launcher.png
new file mode 100644
index 0000000..02e96b9
Binary files /dev/null and b/LonelyTwitter/bin/res/drawable-mdpi/ic_launcher.png differ
diff --git a/LonelyTwitter/bin/resources.ap_ b/LonelyTwitter/bin/resources.ap_
index e136178..3335616 100644
Binary files a/LonelyTwitter/bin/resources.ap_ and b/LonelyTwitter/bin/resources.ap_ differ
diff --git a/LonelyTwitter/gen/ca/ualberta/cs/lonelytwitter/R.java b/LonelyTwitter/gen/ca/ualberta/cs/lonelytwitter/R.java
index 5a80ed6..fb3c93a 100644
--- a/LonelyTwitter/gen/ca/ualberta/cs/lonelytwitter/R.java
+++ b/LonelyTwitter/gen/ca/ualberta/cs/lonelytwitter/R.java
@@ -1,34 +1,34 @@
-/* AUTO-GENERATED FILE.  DO NOT MODIFY.
- *
- * This class was automatically generated by the
- * aapt tool from the resource data it found.  It
- * should not be modified by hand.
- */
-
-package ca.ualberta.cs.lonelytwitter;
-
-public final class R {
-    public static final class attr {
-    }
-    public static final class drawable {
-        public static final int ic_launcher=0x7f020000;
-    }
-    public static final class id {
-        public static final int body=0x7f050003;
-        public static final int clear=0x7f050004;
-        public static final int hello=0x7f050002;
-        public static final int oldTweetsList=0x7f050001;
-        public static final int oldTweetsTitle=0x7f050000;
-        public static final int save=0x7f050005;
-    }
-    public static final class layout {
-        public static final int list_item=0x7f030000;
-        public static final int main=0x7f030001;
-    }
-    public static final class string {
-        public static final int app_name=0x7f040001;
-        public static final int clear=0x7f040003;
-        public static final int hello=0x7f040000;
-        public static final int save=0x7f040002;
-    }
-}
+/* AUTO-GENERATED FILE.  DO NOT MODIFY.
+ *
+ * This class was automatically generated by the
+ * aapt tool from the resource data it found.  It
+ * should not be modified by hand.
+ */
+
+package ca.ualberta.cs.lonelytwitter;
+
+public final class R {
+    public static final class attr {
+    }
+    public static final class drawable {
+        public static final int ic_launcher=0x7f020000;
+    }
+    public static final class id {
+        public static final int body=0x7f050003;
+        public static final int clear=0x7f050004;
+        public static final int hello=0x7f050002;
+        public static final int oldTweetsList=0x7f050001;
+        public static final int oldTweetsTitle=0x7f050000;
+        public static final int save=0x7f050005;
+    }
+    public static final class layout {
+        public static final int list_item=0x7f030000;
+        public static final int main=0x7f030001;
+    }
+    public static final class string {
+        public static final int app_name=0x7f040001;
+        public static final int clear=0x7f040003;
+        public static final int hello=0x7f040000;
+        public static final int save=0x7f040002;
+    }
+}
diff --git a/LonelyTwitter/project.properties b/LonelyTwitter/project.properties
index f049142..c4f09d2 100644
--- a/LonelyTwitter/project.properties
+++ b/LonelyTwitter/project.properties
@@ -8,4 +8,4 @@
 # project structure.
 
 # Project target.
-target=android-10
+target=android-17
diff --git a/LonelyTwitter/src/ca/ualberta/cs/lonelytwitter/AbstractTweet.java b/LonelyTwitter/src/ca/ualberta/cs/lonelytwitter/AbstractTweet.java
new file mode 100644
index 0000000..0a02109
--- /dev/null
+++ b/LonelyTwitter/src/ca/ualberta/cs/lonelytwitter/AbstractTweet.java
@@ -0,0 +1,31 @@
+package ca.ualberta.cs.lonelytwitter;
+
+import java.util.Date;
+
+
+public abstract class AbstractTweet
+{
+	public Date tweetDate;
+	public String tweetBody;
+	
+	public abstract String getTweetBody();

LonelyTwitter/gen/ca/ualberta/cs/lonelytwitter/R.java
/* AUTO-GENERATED FILE.  DO NOT MODIFY.
 *
 * This class was automatically generated by the
 * aapt tool from the resource data it found.  It
 * should not be modified by hand.
 */

package ca.ualberta.cs.lonelytwitter;

public final class R {
    public static final class attr {
    }
    public static final class drawable {
        public static final int ic_launcher=0x7f020000;
    }
    public static final class id {
        public static final int body=0x7f050003;
        public static final int clear=0x7f050004;
        public static final int hello=0x7f050002;
        public static final int oldTweetsList=0x7f050001;
        public static final int oldTweetsTitle=0x7f050000;
        public static final int save=0x7f050005;
    }
    public static final class layout {
        public static final int list_item=0x7f030000;
        public static final int main=0x7f030001;
    }
    public static final class string {
        public static final int app_name=0x7f040001;
        public static final int clear=0x7f040003;
        public static final int hello=0x7f040000;
        public static final int save=0x7f040002;
    }
}
